hack/run_test

```diff
@@ -309,7 +328,7 @@ function test_value_replication() {
     CONTAINER_IP=$slave_ip
     for i in $(seq $max_attempts); do
       result="$(postgresql_cmd -At -c "select * from $table_name")"
```

Comment:
I'm not sure the loop here has some effect.  When the value is not there yet, it is
very likely the $table_name table does not exist yet too.  Then, postgresql_cmd fails
and this would fail the whole command, so no other attempt would me made.

Probably `|| :` is missing.